outputs.tf

```diff
@@ -1,3 +1,3 @@
 output "aspResourceID" {	
-  value = "${azurerm_template_deployment.app_service_plan.outputs["aspResourceID"]}"	
```

Comment:
feel free to try it but we've tried before, if you look at the issue nitin's another one linked to it, normally we bin the plan and rebuild it when this happens